src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -265,7 +265,8 @@ private Task ProcessStreamItem(HubConnectionContext connection, StreamItemMessag
                         for (var parameterPointer = 0; parameterPointer < arguments.Length; parameterPointer++)
                         {
                             if (hubMethodInvocationMessage.Arguments.Length > hubInvocationArgumentPointer &&
-                                descriptor.OriginalParameterTypes[parameterPointer].IsAssignableFrom(hubMethodInvocationMessage.Arguments[hubInvocationArgumentPointer].GetType()))
+                                (hubMethodInvocationMessage.Arguments[hubInvocationArgumentPointer] == null ||
```

Comment:
What if the parameter isn't a nullable type?

Reply:
If I'm not mistaken, json serializer would throw an exception when the parameter is non-nullable and the value is null, and it never reaches this code.